uvo_scraper.rb

```diff
@@ -0,0 +1,41 @@
+require 'curb'
+require 'nokogiri'
+
+class UvoScraper
+
+  SEARCH_URL = 'https://www2.uvo.gov.sk/evestnik?p_p_id=evestnik_WAR_eVestnikPortlets&p_p_lifecycle=1&p_p_state=normal&p_p_mode=view&p_p_col_id=column-2&p_p_col_pos=1&p_p_col_count=2'
+  NEW_ISSUE_URL = 'https://www2.uvo.gov.sk/evestnik/-/vestnik/aktual'
```

Comment:
Freeze mutable objects assigned to constants.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.